core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/output/ToCsvTest.java

```diff
@@ -66,17 +67,20 @@ public void builderShouldCreatePopulatedOperation() {
         assertThat(toCsv.getInput())
                 .hasSize(1);
         assertFalse(toCsv.isIncludeHeader());
-        assertEquals(generator, toCsv.getElementGenerator());
+        assertEquals(generator, toCsv.getCsvGenerator());
+        assertEquals(csvFormat, toCsv.getCsvFormat());
     }
 
     @Test
     @Override
     public void shouldShallowCloneOperation() {
         // Given
         final Entity input = new Entity(TestGroups.ENTITY);
+        final CsvFormat csvFormat = new Neo4jFormat();
         final CsvGenerator generator = new CsvGenerator.Builder().group("group").build();
         final ToCsv toCsv = new ToCsv.Builder()
                 .generator(generator)
+                .csvFormat(csvFormat)
```

Comment:
What happens if you provide a generator AND a format? Which does it use? 
Maybe there should be a test with just generator, just format, then both to define the behaviour?
Maybe it should error when you provide both? Or the generator should overwrite the format?